### PR TITLE
Release v3.0.0

### PR DESCRIPTION
**Pull request by chrmod**
Changes:
* removes blind token generation and injection
* removes API for Ghostery extension telemetry
* removed API for Ghostery browser search engine settings
* removes special manifest for Ghostery browser 
* update for Manifest V3 for Chrome

### CODE REVIEW

File: manifests/chrome.json [hunk 1]
```diff
@@ -1,47 +1,44 @@
 {
   "version": "1.1.0",
```
Comment (smalluban) [inline manifests/chrome.json:2]: Please update version here accordingly here and for Firefox

**Review by smalluban — CHANGES_REQUESTED** [review]
Small fix needed, everything else looks good to me.

File: manifests/chrome.json (hunk 1 above)
Comment (chrmod) [reply manifests/chrome.json:2]: the version is updated by the release script

Comment (smalluban) [reply manifests/chrome.json:2]: So we should remove it from there (as we have in main extension)

**Pull request merged**